client/src/inside/components/page/EhrPageForm.vue

```diff
@@ -47,12 +59,29 @@ export default {
     },
     errors () {
       return this.ehrHelp.getPageErrors(this.formKey)
-    }
+    },
   },
   methods: {
     cssFromDefs: function (element) {
       return element.formCss ? element.formCss : 'noClass'
     },
+
+    promptConfirmDialog : function () {
+      this.$refs.confirmDialog.showDialog(TEXT.TITLE, TEXT.MSG)
+    },
+
+    resetFormData: function () {
+     
+      const children = []
+      this.form.ehr_groups.map(g=>{
+        g.gChildren.map(k=>{
+          children.push(k)
+          return k
+        })
+        return g
+      })
+      this.ehrHelp.resetFormData(children)
```

Comment:
see below and mimic the approach taken to clear table data

Reply:
I had, at first, attempted to do it this way. However, I found a few things that turned out to be different. For example, when trying to get the seed Data from ehrSeed, I have found only the children as an object. And, from my tests so far, I have only found ` `StoreHelper.getSeedEhrData()` as the way to get the seedData from this scope? Could you please give me further guidance on how to proceed? Thank you!

Reply:
Also, `clearTable` just assigns an empty array to the `asLoadedPageData`, as it can be seen here: `asLoadedPageData[tableKey] = []`. The use case is a bit different when it comes to getting the seed vs actually getting the proper array, from my perspective.